spec/models/trans_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Tran, type: :model do
```

Comment:
- [ ] You have done well adding some unit tests for the models, however, you still need to add integrations for different pages in the app.